commit 1022c9627a94c3dca64278641b38176518b1deea (refs/changes/50/2450/4)
Author: Elijah Zupancic <elijah@zupancic.name>
Date:   2017-08-24T12:03:22-07:00 (2 years, 1 month ago)
    
    MANTA-3393 Add Perl CPAN module to SDK list
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/docs/user-guide/sdks.md b/docs/user-guide/sdks.md
index 88dcddd..9f2cc57 100644
--- a/docs/user-guide/sdks.md
+++ b/docs/user-guide/sdks.md
@@ -129,6 +129,16 @@ It has been tested in PHP 5.6, PHP 7.0 and HHVM.
 You can find the source for PHP SDK at [https://github.com/joyent/php-manta](https://github.com/joyent/php-manta).
 
 
+# Perl Module
+
+There is a Perl module available via CPAN that supports object operations. It is
+a community driven module, so updates to it are dependent upon community
+involvement.
+The module is available on [CPAN](http://search.cpan.org/~andrewh/Manta-Client/).
+
+You can find the source for the Perl module at [https://github.com/joyent/Manta-Client](https://github.com/joyent/Manta-Client).   
+
+
 # Golang SDK
 
 The Go SDK for Manta is currently in active development. It is a combination SDK
